### PR TITLE
Blog Post: CSV/Email to Vendia Share

### PR DESCRIPTION
**Pull request by caroll-vendia**
First part of a series about integrating data with Vendia Share.

### CODE REVIEW

File: posts/2022-03-15-add-csv-to-vendia-share.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+There are numerous examples partner data sharing in which one partner’s data does not need to be as 'real-time' or as sophisticated as the rest of the partner network - and often the simplest solution, like adding a CSV, is the best solution. 
+
+## Step-by-step how to add a CSV to Vendia Share
+
+This example will demonstrate how to send scalar data from a comma-separated value (CSV) file to a [Vendia Share Uni](https://vendia.net/docs/share/dev-and-use-unis).  
+We will deploy the example using the [Vendia Share Command Line Interface (CLI)](https://vendia.net/docs/share/cli) and the [AWS Serverless Application Model (SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html) to send scalar data from a CSV to a Vendia Share Uni.  
+Serverless resources like a [AWS S3](https://aws.amazon.com/s3/) bucket and [AWS Lambda](https://aws.amazon.com/lambda/) function will be deployed.  CSV processing with a AWS Lambda function will be triggered by uploading data to a AWS S3 bucket.  The Lambda function will read through the CSV and publish data to a node's GraphQL endpoint.
```
Comment (brian-vendia) [inline posts/2022-03-15-add-csv-to-vendia-share.md:27]:
```suggestion
Serverless resources like a [AWS S3](https://aws.amazon.com/s3/) bucket and [AWS Lambda](https://aws.amazon.com/lambda/) function will be deployed.  CSV processing with a AWS Lambda function will be triggered by parsing an email message and uploading CSV attachments to a AWS S3 bucket.  The Lambda function will read through the CSVs and publish data to a node's GraphQL endpoint.
```

**Review by brian-vendia — CHANGES_REQUESTED** [review]
Minor changes - some of which need to be reflected in the Github example we reference (that's on me).

**Comment by brian-vendia** [comment]
@DavidWells The [blog preview page](https://staging-preview-caroll-vendia-csv-blog-post--vendia-site.netlify.app/blog) has a broken link, but it is a valid link (https://github.com/vendia/examples/blob/main/share/email-to-share/img/email-to-share.png). Is this an artifact of the build process, or is this something to do with the link itself?

**Review by caroll-vendia — COMMENTED** [review]
Approve all changes

**Pull request merged**